### PR TITLE
user login, logout, edit profile

### PR DESCRIPTION
**Pull request by tranduytung**

### CODE REVIEW

File: app/views/users/registrations/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+      <%= f.label :name %><br />
+      <%= f.text_field :name, autofocus: true, class: "form-control" %>
+
+      <%= f.label :email %><br />
+      <%= f.email_field :email, autofocus: true, class: "form-control"  %>
+
+      <%= f.label :password %>
+      <% if @minimum_password_length %>
+      <em>(<%= @minimum_password_length %> characters minimum)</em>
```
Comment (nguyenngocha) [inline app/views/users/registrations/new.html.erb]: <%= "(#{@minimum_password_length} characters minimum)" if @minimum_password_length %>

**Comment by nguyenngocha** [comment]
OK

**Comment by daonhat** [comment]
OK

File: app/views/users/registrations/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      <%= f.password_field :password_confirmation, autocomplete: "off",
+        class: "form-control" %>
+
+      <%= f.label :current_password %>
+      <i><%= t :current_password %></i><br/>
+      <%= f.password_field :current_password, autocomplete: "off",
+        class: "form-control" %>
+
+      <div class="actions">
+        <%= f.submit t("user.update_profile"), class: "btn-primary" %>
```
Comment (kagoromo) [inline app/views/users/registrations/edit.html.erb]: class: "btn btn-primary"

File: app/views/users/registrations/edit.html.erb [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      </div>
+    <% end %>
+    <%= link_to t(:back), :back %>
```
Comment (kagoromo) [inline app/views/users/registrations/edit.html.erb]: <%= link_to t(:back), :back, class: "btn btn-default col-md-4 pull-left" %>

**Comment by kagoromo** [comment]
OK

**Pull request merged**